src/features/questions/questions-list/FilteredQuestionGroups.tsx

```diff
@@ -21,6 +22,41 @@ const FadeInQuestionGroup = withFadeIn<QuestionGroupItemProps>(
 export const FilteredQuestionGroups: React.FC<{
   filterQuestionGroups: ReadonlyArray<QuestionGroup>
 }> = ({ filterQuestionGroups }) => {
+  const [focusedItem, setFocusedItem] = useState(1)
+  const [focusedInsideItem, setFocusedInsideItem] = useState(0)
+
+  const isUpPressed = useKeyIsPressed(Keys.Up)
+  const isDownPressed = useKeyIsPressed(Keys.Down)
+  const isEnterPressed = useKeyIsPressed(Keys.Enter)
+  const isOptionPressed = useKeyIsPressed(Keys.Option)
+  const isEscapePressed = useKeyIsPressed(Keys.Escape)
+
+  useEffect(() => {
+    const length = filterQuestionGroups.length
+
+    if (!focusedInsideItem && isDownPressed && focusedItem < length) {
+      setFocusedItem((prev) => prev + 1)
+    } else if (!focusedInsideItem && isUpPressed && focusedItem > 1) {
+      setFocusedItem((prev) => prev - 1)
+    } else if (!focusedInsideItem && isUpPressed && focusedItem === 1) {
+      setFocusedItem(length)
```

Comment:
A bit confusing if it starts with 1 or what do you think?
```suggestion
      setFocusedItem(length - 1)
```